src/components/fields/ArrayField.js

```diff
@@ -101,7 +101,7 @@ class ArrayField extends Component {
       event.preventDefault();
       this.asyncSetState({
         items: this.state.items.filter((_, i) => i !== index)
-      }, {validate: true}); // refs #195
```

Comment:
As you can see here, the forced validation option is used to address #195. Your patch reintroduces this bug.